examples/pure/number_input/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "pure_number_input"
+version = "0.1.0"
+authors = ["leang27 <52003343+leang27@users.noreply.github.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+iced_aw = { path = "../../..", default-features = false, features = [
+    "number_input",
+    "icons",
+    "pure",
+] }
+
+[dependencies.iced]
+git = "https://github.com/iced-rs/iced.git"
+#branch = "generic-widget-tree"
```

Comment:
Iced in your example is not up to date with the rev used in the rest of the project.